### PR TITLE
Version2.0.0

### PR DESCRIPTION
**Pull request by kd4001**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,24 +24,28 @@
...
+    "@date-io/core": "^1.3.6"
   },
   "devDependencies": {
-    "bs-platform": "^7.0.1",
+    "bs-platform": "^8.2.0",
     "reason-react": "^0.9.0"
   },
   "peerDependencies": {
-    "bs-platform": "^7.0.1",
+    "bs-platform": "^8.2.0",
```
Comment (Prateek13727) [inline package.json:47]: the version in Spot-CRM is not the same as this. Also, the latest version is  8.4.2

Comment (Prateek13727) [reply package.json:47]: can you upgrade reason-react to 0.9.1 in peer-dependencies ?

File: src/components/generic/Popover/PopoverComponent.re [hunk 2]
```diff
@@ -1,36 +1,70 @@
...
+      ~vertical=vertical_transform->mapVertical->MaterialUi_Popover.Vertical.enum,
+      (),
     );
 
   MaterialUi.(
     <Popover
       anchorOrigin=position
       transformOrigin=transform
-      open_={Option.isSome(anchor_el)}
+      _open={Option.isSome(anchor_el)}
```
Comment (Prateek13727) [inline src/components/generic/Popover/PopoverComponent.re:59]: to discuss ?

File: src/components/generic/icons/Icon.re [hunk 3]
```diff
@@ -38,3 +38,9 @@ module EditIcon =
     [@bs.module "@material-ui/icons/EditOutlined"]
     external reactClass: React.component('a) = "default";
   });
+
+module MoreVertIcon =
```
Comment (Prateek13727) [inline src/components/generic/icons/Icon.re:42]: where is this being used ?

File: src/components/generic/Popover/PopoverComponent.re (hunk 2 above)
Comment (kd4001) [reply src/components/generic/Popover/PopoverComponent.re:59]: Changes in binding

File: src/components/generic/icons/Icon.re (hunk 3 above)
Comment (kd4001) [reply src/components/generic/icons/Icon.re:42]: Order Map three dots symbol, MenuList.re

**Pull request merged**